WordPress/src/test/java/org/wordpress/android/push/NotificationHelperTest.kt

```diff
@@ -0,0 +1,114 @@
+package org.wordpress.android.push
+
+import android.os.Bundle
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.apache.commons.text.StringEscapeUtils
+import org.junit.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import org.wordpress.android.BaseUnitTest
+import org.wordpress.android.models.Note
+import org.wordpress.android.push.GCMMessageHandler.PUSH_TYPE_COMMENT
+import org.wordpress.android.push.GCMMessageHandler.PUSH_ARG_MSG
+import org.wordpress.android.push.GCMMessageHandler.PUSH_ARG_TITLE
+import org.wordpress.android.push.GCMMessageService.PUSH_ARG_NOTE_ID
+import org.wordpress.android.ui.notifications.SystemNotificationsTracker
+import org.wordpress.android.ui.notifications.utils.NotificationsUtilsWrapper
+import kotlin.test.assertEquals
+
+private const val PUSH_TYPE_NOTE_A_COMMENT = "NotAComment"
```

Comment:
Should the variable name be `PUSH_TYPE_NOT_A_COMMENT`? (I.e. is this a typo?)

```suggestion
private const val PUSH_TYPE_NOT_A_COMMENT = "NotAComment"
```

Reply:
Good catch @mkevins 👍 
Pushed a fix with 7929dacb4227511a8a7f7d317a0eef6fffbc2424